src/main/kotlin/app/revanced/patches/spotify/audio/resource/patch/DisableCaptureRestrictionResourcePatch.kt

```diff
@@ -0,0 +1,36 @@
+package app.revanced.patches.spotify.audio.resource.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.impl.ResourceData
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.impl.ResourcePatch
+import app.revanced.patches.spotify.audio.annotation.DisableCaptureRestrictionCompatibility
+import app.revanced.patches.youtube.misc.manifest.patch.FixLocaleConfigErrorPatch
+import org.w3c.dom.Element
+
+@Name("disable-capture-restriction-resource-patch")
+@DependsOn([FixLocaleConfigErrorPatch::class])
```

Comment:
Is this really necessary?

Reply:
Removed it, that was a left-over from another ResourcePatch that I used for reference